CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.13.4 FATAL_ERROR)
 project(Thorin)
 
 set(PACKAGE_VERSION "0.3.9")
-set(CMAKE_CONFIGURATION_TYPES "Debug;Release" CACHE STRING "limited config" FORCE)
+#set(CMAKE_CONFIGURATION_TYPES "Debug;Release" CACHE STRING "limited config" FORCE)
```

Comment:
What's that about ?

Reply:
Never understood why it was there the first place. Usually the default are "Debug;MinSize;RelWithDebug;Release". But we limited ourselfs to Debug and Release. So happy to see it go finally. :D